### PR TITLE
Migrate from gorp to sqlx in store/sqlstore/command_webhook_store.go

### PR DESCRIPTION
**Pull request by spinales**
Fixes https://github.com/mattermost/mattermost-server/issues/18499

### CODE REVIEW

File: store/sqlstore/command_webhook_store.go [hunk 1]
```diff
@@ -47,7 +47,10 @@ func (s SqlCommandWebhookStore) Save(webhook *model.CommandWebhook) (*model.Comm
 		return nil, err
 	}
 
-	if err := s.GetMaster().Insert(webhook); err != nil {
+	if _, err := s.GetMasterX().NamedExec(`INSERT INTO WebHook
```
Comment (agnivade) [inline store/sqlstore/command_webhook_store.go:50]: `INSERT INTO CommandWebhooks`

Also `CreatedAt` should be `CreateAt`

File: store/sqlstore/command_webhook_store.go [hunk 2]
```diff
@@ -70,7 +73,7 @@ func (s SqlCommandWebhookStore) Get(id string) (*model.CommandWebhook, error) {
 		return nil, errors.Wrap(err, "get_tosql")
 	}
 
-	if err := s.GetReplica().SelectOne(&webhook, queryString, args...); err != nil {
+	if err := s.GetReplicaX().Select(&webhook, queryString, args...); err != nil {
```
Comment (agnivade) [inline store/sqlstore/command_webhook_store.go:76]: This should be `.Get`

**Review by agnivade — COMMENTED** [review]
Thank you @spinales ! Just some small things to fix.

**Comment by spinales** [comment]
> Thank you @spinales ! Just some small things to fix.

Sorry for that, I missed that part there 😅

File: store/sqlstore/command_webhook_store.go (hunk 1 above)
Comment (spinales) [reply store/sqlstore/command_webhook_store.go:50]: sorry for that, i'm habit to put the verb in past

**Review by agnivade — APPROVED** [review]
Thank you @spinales !

**Comment by spinales** [comment]
Thanks to you for the opportunity 😄

**Comment by emilyacook** [comment]
Thanks for participating in Hacktoberfest! You can claim your sticker set here: https://get.printfection.com/hacktober21/4144583267 @spinales

**Pull request merged**